internal/controller/drplacementcontrol.go

```diff
@@ -339,11 +339,16 @@ func (d *DRPCInstance) RunFailover() (bool, error) {
 	const done = true
 
 	if d.instance.Spec.FailoverCluster == "" {
-		msg := "missing value for spec.FailoverCluster"
-		addOrUpdateCondition(&d.instance.Status.Conditions, rmn.ConditionAvailable, d.instance.Generation,
-			d.getConditionStatusForTypeAvailable(), string(d.instance.Status.Phase), msg)
-
-		return done, fmt.Errorf(msg)
+		addOrUpdateCondition(
+			&d.instance.Status.Conditions,
+			rmn.ConditionAvailable,
+			d.instance.Generation,
+			d.getConditionStatusForTypeAvailable(),
+			string(d.instance.Status.Phase),
+			"missing value for spec.FailoverCluster",
+		)
```

Comment:
Formatting change is good but not related to the fix. Repeating the error message instead of reusing it is not an improvement.

Reply:
The formatting change is due to line length, due to adding the string instead of `msg`, hence is related.

The condition message and error are now separate (though the same string). Each can change as appropriate IMO.

Reply:
The reported issue by the linter is as follows:

```
internal/controller/drplacementcontrol.go:346:27: printf: non-constant format string in call to fmt.Errorf (govet)
		return done, fmt.Errorf(msg)
		                        ^
```

If we need the retain the same string, an alternate fix is as follows:
```
		const msg = "missing value for spec.FailoverCluster"

		addOrUpdateCondition(&d.instance.Status.Conditions, rmn.ConditionAvailable, d.instance.Generation,
			d.getConditionStatusForTypeAvailable(), string(d.instance.Status.Phase), msg)

		return done, fmt.Errorf(msg)
```

Is this what you prefer?

Reply:
Made the above change as it is the most minimal at present, that go around in circles on nits...